.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 ignore =
     E203,
-    W503, # Incompatible with black see https://github.com/psf/black/issues/315
```

Comment:
It was nice to be able to comment about why something was disabled, but it's not possible anymore...
```
flake8/options/config.py", line 131, in parse_config

    raise ValueError(

ValueError: Error code '#' supplied to 'ignore' option does not match '^[A-Z]{1,3}[0-9]{0,3}$'
```

Reply:
I know you're curious Daniel, so I took the time to see what the flake8 maintainer was saying about it, inline comment are not possible anymore, and it's not up for debate as expected :) But we could comment the whole line apparently.

Reply:
That's too bad!